tests/TestCase/I18n/I18nTest.php

```diff
@@ -167,6 +167,12 @@ public function testPluginMesagesLoad(): void
             'String 1 (from plugin three)',
             $translator->translate('String 1')
         );
+
+        $translator = I18n::getTranslator('company/test_plugin_three.custom');
```

Comment:
It would be good to have an integration test with `__d()` covering the userland use case as well.